securedrop/client.py

```diff
@@ -167,6 +170,25 @@ async def add_contact(self):
         if msg != "":
             print("Failed to add contact: ", msg)
 
+    async def list_contacts(self):
+        msg = ""
+        try:
+            with open(TEST_FILENAME, 'w') as f:
+
+                await self.write(bytes(ListContactsPackets()))
+                contact_dict = ListContactsPacketsResponse(data=(await self.read())[4:]).contacts
+                contacts_keys = list(contact_dict.keys())
+                json.dump(contact_dict, f)
+                # print contacts by Email and Name
+                print("Email:\t\t\t\tName:")
```

Comment:
You should check if the size is zero, and print something like "No contacts online" instead if it is the case.

Reply:
Changed.